static/js/impala/promos.js

```diff
@@ -31,6 +31,16 @@ function initPromos($context, module_context, version, platform) {
             $promos.trigger('promos_shown', [$promos]);
             $('.persona-preview', $promos).previewPersona(true);
         }
+        // Hide panel is we have no promos to show
+        if (resp !== '' && $('body#restyle').length === 1) {
+            $('#page > .secondary').addClass('shift-secondary');
+            $('#background-wrapper').addClass('carousel-header');
+            $('#promos').addClass('show');
+            $('#side-nav').addClass('expanded');
+            $('#background-wrapper').css('height',
+                $('.amo-header').height() +
+                $('#promos').height() + 40 + 'px');
```

Comment:
`height()` assumes 'px' if the value is a `Number` [1] which would help make this more direct e.g. rather than concat of numbers and a string.

[1] https://api.jquery.com/height/#height-value